### ISSUE TITLE
`name-replacements`: don't suggest reserved keywords

### ISSUE THREAD

**Issue by fregante** [sindresorhus/eslint-plugin-unicorn#3715]
### Description

Since the rule is no longer called `prevent-abbreviation`, I think suggesting names like `arguments_` and `function_` should be undone. Every time I enabled the rule in repos there's been a huge pushback particularly on ugly lengthenings like this one. 

I love that the feature reports `e` and `evt` but certain replacements are very hard to like.

### Examples

```js
// ✅
function foo(...args) {
	console.log(args)
}
```

**Comment by sindresorhus** [comment]
> I think suggesting names like arguments_ and function_ should be undone.

I disagree. I prefer the consistency of not using abbreviations over a slight ugliness of appending an underscore.

People are always going to dislike new things. Just because people are used to `args` doesn't mean it's any better or worse. It's just different. When promises were new, people thought to keep callbacks. Same with async/await over promises.

Closing as I don't plan to change this. But you can change it yourself in the config.
